test/ig/IGExporter.menu-xml.test.ts

```diff
@@ -30,14 +30,15 @@ describe('IGExporter', () => {
       temp.cleanupSync();
     });
 
-    it('should do nothing when config.menu is undefined and none provided', () => {
+    it('should log a warning when config.menu is undefined and none provided', () => {
       const pkg = new Package(minimalConfig);
       const igDataPath = path.resolve(__dirname, 'fixtures', 'simple-ig');
       const exporter = new IGExporter(pkg, null, igDataPath);
       exporter.addMenuXML(tempOut);
       const menuPath = path.join(tempOut, 'input', 'includes', 'menu.xml');
       expect(fs.existsSync(menuPath)).toBeFalsy();
-      expect(loggerSpy.getAllMessages()).toHaveLength(0);
+      expect(loggerSpy.getAllMessages('warn')).toHaveLength(1);
+      expect(loggerSpy.getLastMessage()).toMatch(/No "menu" property or file was found\./s);
```

Comment:
Since you include some variables in the message (input folder and config file name), I think it would be good to make sure those are coming through correctly too. I think you can just add something like `.*input[/\]includes folder.* or .* in sushi-config\.yaml` to the regex.